src/helpers/validate.js

```diff
@@ -18,9 +19,22 @@ function validateFileNamePath(path) {
   return mask.test(path)
 }
 
+/**
+ * Validate that a SHA is the correct length and content
+ * @param {string} commitSHA
+ * @param {number} requestedLength
+ * @returns {boolean}
+ */
+function validateSHA(commitSHA, requestedLength) {
```

Comment:
my JS is trash, is there a way to mark `requestedLength` as optional?

Reply:
Yes. What do you want for the default? 40?

Reply:
> 40?

For reference, full-length Git SHAs are 40 characters long (40 hexidecimal characters which, translated to ASCII or UTF-8, etc., equates to 40 characters).

```suggestion
const GIT_SHA_LENGTH = 40;

function validateSHA(commitSHA, requestedLength = GIT_SHA_LENGTH) {
```